src/yesql/generate.clj

```diff
@@ -126,7 +128,7 @@
                                                                                       required-args))]
                                                              {:keys as-vec}
                                                              {})
-                                                global-args {:keys ['connection]}]
+                                                global-args {:keys ['connection 'identifiers 'row-fn 'result-set-fn]}]
```

Comment:
I see the argument for a global `:identifiers` arg, but not `row-fn/result-set-fn`...?